### DISCUSSION TITLE
Proof of Stake

### DISCUSSION THREAD

**Discussion by umikali222** [dogecoin/dogecoin#3110] [Q&A]
Will Dogecoin Core 1.14.7 use Proof of Stake?

**Comment by patricklodder** [comment]
Extremely unlikely, because:

There currently is no code proposing this and there has been no real discussion of any proposed implementation which is [required per the contribution guidelines](https://github.com/dogecoin/dogecoin/blob/3a29ba6d497cd1d0a32ecb039da0d35ea43c9c85/CONTRIBUTING.md?plain=1#L142-L150). Both the consensus change discussion and subsequent review and testing will take a lot of time. I'm currently working through a list of issues that may be urgent enough to release 1.14.7 sooner rather than later. Therefore I expect that even if between me starting to write this reply and hitting the "answer" button, someone would propose the code, 1.14.7 will still be released without it.

Also please note that I am trying to make it so that 1.21 becomes a much better target for consensus changes than 1.14, so that we can omit consensus changes from 1.14 unless they are a backport of what gets proposed and accepted in 1.21, for backward compatibility.

**Reply by umikali222** [reply]
Thank you for your reply. I heard that work on proof of stake would start in 2022. So has it already started? If not when will it start? When should Dogecoin switch to PoS?

**Reply by tealethetoolman** [reply]
You are free to make a branch with changes that move to PoS and submit a PR. But since PoS is insecure and removes equitable distribution to a more centralized model, the adoption of your branch can predictably be minimal.

**Reply by patricklodder** [reply]
>  I heard that work on proof of stake would start in 2022. 

Who said that? Sounds a bit crazy. The only thing I have seen besides banter was a draft document that proposed the formation of a private commission that would design something, which I would not support because it reeks of authoritarianism by the people proposing it and them trying to control Dogecoin. That's in my opinion not what a "people's coin" should be built upon. If someone has actual ideas, just present them. We can always discuss it.

Also, as @tealethetoolman points out, there are real hurdles that must be overcome for it to even be feasible. Currently 2 parties have over 40% of all DOGE in custody combined. PoS would mean they would get a veto. I cannot imagine that anyone thinks that this would be a beneficial change to a permissionless network. These kinds of things must be addressed, risk assessments must be made.

**Comment by paulVu** [comment]
I hear alot about Proof of Stake for DOGE, I think DOGE just keep going to POW for more secure, low fee, fast transaction, easy to build function add on.
